reflections_test.go

```diff
@@ -443,6 +443,33 @@ func TestItems_deep(t *testing.T) {
 	assert.Equal(t, itemsDeep["Number"], 17)
 }
 
+func TestGetNameFieldByTag(t *testing.T) {
+
+	dummyStruct := TestStruct{
+		Dummy: "test",
+		Yummy: 123,
+	}
+
+	tagJson := "dummytag"
+	field, err := GetNameFieldByTag(dummyStruct, tagJson, "test")
+
+	assert.NoError(t, err)
+	assert.Equal(t, field, "Dummy")
+}
+
+func TestGetNameFieldByTag_on_non_existing_tag(t *testing.T) {
```

Comment:
This test verifies that looking for an existing struct tag, with a non-existing matching value, fails. Could you also add one verifying that looking a for a non-existing tag fails, please? 🙇🏻